### ISSUE TITLE
Python 3 print error

### ISSUE THREAD

**Issue by Paul-dH** [euangoddard/hr.py#3]
Hi!

Nice tool :) Thanks!
I do get a error at [line 41](https://github.com/euangoddard/hr.py/blob/e9a8081e37cf315e3781dab905e194498a07b048/hr.py#L41), this is the print line. Python 3 complains that it should be in parentheses.

The error:
```
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(output[:cols])?
```

I've tested the change below and that works:
```python
print(output[:cols])
```

Regards, Paul

**Comment by euangoddard** [comment]
Do you have the latest version checked out? Looking at the source code it appears that the syntax matches what you've suggested.

Did you install from PyPi? It could be that I never published the latest package

**Comment by Paul-dH** [comment]
I did a pip3 install hr, I do realise I only checked the source of the installed version om my pc and the code in the github repro. I didn't checked the version on PyPi...


Van: Euan Goddard
Verstuurd: dinsdag 4 september 10:03
Onderwerp: Re: [euangoddard/hr.py] Python 3 print error (#3)
Aan: euangoddard/hr.py
Cc: Paul den Hertog, Author


Do you have the latest version checked out? Looking at the source code it appears that the syntax matches what you've suggested.
Did you install from PyPi? It could be that I never published the latest package

**Comment by euangoddard** [comment]
I will try to get round to updating the package on PyPI this evening. Thanks for the spot!

**Comment by Paul-dH** [comment]
Cool, no problem!
________________________________

**Comment by euangoddard** [comment]
I'm having real trouble uploading a new distribution to PyPI. It has changed a lot since I last used it and I no longer do Python development professionally so am a bit out of touch. Any suggestions?

**Comment by user2589** [comment]
@euangoddard 
How to upload to PyPI:
- install [twine](https://pypi.org/project/twine/): `pip install twine`
- build package: `python setup.py sdist bdist_wheel`
- upload: `twine upload dist/*` (will ask for PyPI account credentials)

Hope it helps

**Comment by tddschn** [comment]
It seemed that the author wasn't able to upload the latest version to PyPI,

so I made a fork with python3 support and properly configured entry point here: 

https://github.com/tddschn/hr-tddschn, PyPI: https://pypi.org/project/hr-tddschn/

Install section from the README:

```
$ pip install hr-tddschn
```

Or, if you only want to use it as a CLI app:
```
$ pipx install hr-tddschn
```

Then run it with `hr`.

Hope this helps!

**Comment by euangoddard** [comment]
Apologies for not actioning this myself - must have missed the response - I can look into sorting this for the package as is

**Comment by euangoddard** [comment]
I've published the Python3 version as v0.2. Hope this helps
